test-requirements.txt

```diff
@@ -1,2 +1,3 @@
 nose
 mock
+flask
```

Comment:
Maybe? I've found that to actually be useful in a number of scenarios to help detect when Flask makes breaking changes so that we can address those incompatibilities.

In many projects, I've seen the dep left without a version range and then, if the case arises where the tests are failing because of a new flask version causing breakage and its blocking something else from going out, we can add the range at that time.

But I don't feel really strongly about it; just sharing my experience from a couple other Python projects.